src/Elastic.Apm/Report/Serialization/BodyStringSanitizerConverter.cs

```diff
@@ -22,18 +22,23 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
 		{
 			if (value is string strValue)
 			{
-				var formValues = strValue.Split('&');
-
 				if (!FormatCheck(strValue))
 					writer.WriteValue(strValue);
 				else
 				{
+					var formValues = strValue.Split('&');
+
 					var sb = new StringBuilder();
 					foreach (var formValue in formValues)
 					{
 						var formsValueSplit = formValue.Split('=');
-						if (formsValueSplit.Length != 2)
-							continue;
+						if (formsValueSplit.Length != 2 || string.IsNullOrWhiteSpace(formsValueSplit[0])
```

Comment:
`WhiteSpace` part of `IsNullOrWhiteSpace` will cause empty strings and strings white space only to be exposed instead of being replaced by "[REDACTED]".

Reply:
Changed it.